.github/workflows/push_openapi_spec_to_readme.yml

```diff
@@ -5,6 +5,13 @@ on:
     branches:
       - master
 
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  id-token: write
+
 jobs:
```

Comment:
apologies, I also realized in one of my own PRs that I was wrong about the secret name to fetch from. left a comment here: https://github.com/Asana/openapi/pull/77/files#r1560120151

you can add that fix to this PR if you want!

Reply:
Yup I saw that and sent a commit